pants.toml

```diff
@@ -173,3 +173,6 @@ config = "@build-support/regexes/config.yaml"
 
 [toolchain-setup]
 repo = "pants"
+
+[buildsense]
+enable = false
```

Comment:
Nit: if you're making another change already, would be good to add a blank line.